llvm/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
 # See docs/CMake.html for instructions about how to build LLVM with CMake.
 
-cmake_minimum_required(VERSION 3.4.3)
+cmake_minimum_required(VERSION 3.14)
 
-if ("${CMAKE_VERSION}" VERSION_LESS "3.13.4")
+if ("${CMAKE_VERSION}" VERSION_LESS "3.14")
   message(FATAL_ERROR
     "Your CMake version is ${CMAKE_VERSION}. The minimum version of CMake "
-    "required to build LLVM is now 3.13.4.")
+    "required to build LLVM is now 3.14")
 endif()
```

Comment:
This seems to be dead code now.

Reply:
I don't quite understand the comment - please elaborate.

Reply:
OK, reverted changes in the dead code.